crates/flowctl/src/pagination.rs

```diff
@@ -0,0 +1,80 @@
+use std::marker::PhantomData;
+
+use page_turner::PageTurner;
```

Comment:
Feel free to ignore this comment since I'm not 100% confident here, but it _feels_ like this library may not be pulling its weight vs a hand written implementation. Up to you whether that's really the case, but just thought I'd mention it in case you were on the fence.

Reply:
Yeah, I did think about writing my own here. But looking at the implementation of PageTurner, it's actually doing a lot for us that I wouldn't want to re-implement. I think ideally the way this would work is we'd implement `PageTurner` for `postgrest::Builder`, but we can't do that so we have this somewhat awkward `PaginationClient` instead.

I just added `into_items()` so we don't have to deal with any of the pagination stuff directly, instead just get a stream of rows.